generic-template/node/src/service.rs

```diff
@@ -151,7 +173,7 @@ async fn start_node_impl(
     let backend = params.backend.clone();
     let mut task_manager = params.task_manager;
 
-    let (relay_chain_interface, collator_key) = build_relay_chain_interface(
+    let interface = build_relay_chain_interface(
```

Comment:
maybe it requires a new type/struct declaration for this with the code changes, but I think it will worth it instead of error prone `.0` and `.1` syntax